src/app/shared/form/form.component.spec.ts

```diff
@@ -359,7 +359,7 @@ describe('FormComponent test suite', () => {
       expect(formComp.submitForm.emit).toHaveBeenCalled();
     });
 
-    it('should not emit submit Event on form submit whether the form is not valid', () => {
+    it('should not emit submit Event on Submit whether the form is not valid', () => {
```

Comment:
This maybe is a search/replace error, too.

Reply:
thanks, fixed today